vscode-client/package.json

```diff
@@ -26,7 +26,19 @@
     "onLanguage:shellscript"
   ],
   "main": "./out/src/extension",
-  "contributes": {},
+  "contributes": {
+      "configuration": {
+          "type": "object",
+          "title": "Bash IDE configuration",
+          "properties": {
+              "bash.explainshellEndpoint": {
```

Comment:
Let us call this `bashIde.` instead of `bash.`

Reply:
Done